rjsone/def.bzl

```diff
@@ -5,9 +5,35 @@ Defines custom build rules that allow to use rjsone.
 _rjsone_binary_label = "@com_github_wryun_rjsone//:rjsone"
 _runner_binary_label = "@com_github_atlassian_bazel_tools//rjsone:runner"
 
+def _keyed_raw_values_to_args(keyed_raw_values):
+    return [
+        "%s::+%s" % (key, value)
+        for key, value in keyed_raw_values.items()
+    ]
+
+def _keyed_yaml_values_to_args(keyed_yaml_values):
+    return [
+        "%s:+%s" % (key, value)
+        for key, value in keyed_yaml_values.items()
+    ]
+
+def _keyed_contexts_to_args(keyed_contexts):
+    args = []
+    for context_target, context_key in keyed_contexts.items():
+        files = context_target.files.to_list()
+        if len(files) == 0:
+            fail("Target %s produces no files" % context_target.label, attr = "keyed_contexts")
+        elif len(files) == 1:
+            args.append("%s:%s" % (context_key, files[0].path))
+        else:
+            args.append("%s:.." % context_key)
```

Comment:
I have the vague feeling we use '...' (which gets the filename).

Reply:
I'd like to get this merged - it needs more work anyway. `:...` support if we need it, stamping (e.g. depending on things like git commit, build timestamp which we also use) and possibly more things. I'll send more PRs =)

Reply:
I.e. what we have here does work, but it gets to big to review.